libraries/AP_HAL_ChibiOS/hwdef/scripts/STM32L431xx.py

```diff
@@ -31,7 +31,8 @@
     # flags of 2 means faster memory for CPU intensive work
     'RAM_MAP' : [
         # use 2nd mapping of SRAM2 to allow a single memory space
-        (0x20000000, 64, 1), # SRAM1/SRAM2
+        (0x20000000, 48, 0), # SRAM1
```

Comment:
please check if it works without this change

Reply:
may need ALT_RAM_MAP

Reply:
Removed